package.json

```diff
@@ -49,9 +49,9 @@
     "typescript": "^4.9.3"
   },
   "peerDependencies": {
-    "@types/react": ">=17 <= 18",
+    "@types/react": ">=16 <= 18",
```

Comment:
Technically, this should be 16.8 because we use hooks.

Reply:
Oh good call. That would make sense why other packages mention 16.8